src/ValidationRules.jsx

```diff
@@ -42,6 +42,9 @@ const validations = {
     maxNumber: (value, max) => !isExisty(value) || isEmpty(value) || parseInt(value, 10) <= parseInt(max, 10),
 
     minNumber: (value, min) => !isExisty(value) || isEmpty(value) || parseInt(value, 10) >= parseInt(min, 10),
+
+    minLength: (value, length) => !isExisty(value) || isEmpty(value) || value.length >= length,
+    maxLength: (value, length) => !isExisty(value) || value.length <= length
```

Comment:
What is `length`? String length? If yes, please change names to `minStringLength` and add check that passed `value` is string